explanations/EXPO_INTEGRATION.md

```
@@ -0,0 +1,206 @@
+| EAS Build | ❌ | ✅ | ✅ |
+| Max Builds/Day | 5 | 50 | 500 |
+| Emulator Minutes/Day | 0 | 120 | 600 |
+
+## Environment Variables
+
+For EAS Build support, add to `.env`:
+```bash
+EXPO_ACCESS_TOKEN=your_expo_token_here
+```
+
+Get your token from: https://expo.dev/settings/access-tokens
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)